unit_tests/multiphase/test_vof_cons.cpp

```diff
@@ -253,7 +253,25 @@ class VOFConsTest : public MeshTest
         }
 
         if (dir >= 0) {
-            check_accuracy(dir, m_nx, tol, vof);
+            // Create scratch field to store error
+            auto error_ptr = repo.create_scratch_field(1, 0);
+            auto& error_fld = *error_ptr;
+            // Initialize at 0
+            for (int lev = 0; lev < repo.num_active_levels(); ++lev) {
+                error_fld(lev).setVal(0.0);
+            }
+
+            get_accuracy(error_fld, dir, m_nx, vof);
+
+            // Check error in each mfab
+            constexpr amrex::Real vofsol_check = 0.0;
+            for (int lev = 0; lev < repo.num_active_levels(); ++lev) {
+                // Sum error for each component individually
+                amrex::Real err_lev = error_fld(lev).max(0);
+                // Check error
+                EXPECT_NEAR(err_lev, vofsol_check, tol);
+                err_lev = error_fld(lev).max(1);
```

Comment:
missing an expect near here? also could make these checks (and some a bit above in vof) one-liners

Reply:
unintended max - I only have one component of error en this test. Maybe this is why the unit tests fail in the CI?